Tests/PropertyListEncoding.swift

```diff
@@ -0,0 +1,67 @@
+import Alamofire
+
+internal struct PropertyListEncoding: ParameterEncoding {
```

Comment:
Since this is internal, why all the properties and methods are public?

Reply:
ah good catch, I just copied the encoding for testing as Alamofire removed it from version 5 😮